bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperClientStats.java

```diff
@@ -67,6 +67,7 @@
     String CHANNEL_START_TLS_OP = "START_TLS";
     String CHANNEL_TIMEOUT_START_TLS_OP = "TIMEOUT_START_TLS";
     String NETTY_EXCEPTION_CNT = "NETTY_EXCEPTION_CNT";
+    String CLIENT_SEND_WAIT_TIMER = "CLIENT_SEND_WAIT_TIMER";
```

Comment:
we don't really use "SEND" as the term at the project. It has to be either add or read.

Reply:
@sijie this is not actually add or read. This is time for either add or read request when request send is blocked.
I am open for suggestions of better name, would CLIENT_NETTY_WAIT_TIMER or CLIENT_WRITE_WAIT_TIMER sound better? though the wait is in ledger handle. CLIENT_PCBC_WRITE_WAIT_TIMER might be better. though "write" can be confused with add op.

Reply:
How about CHANNEL_WRITE?